### DISCUSSION TITLE
Auto Login not working?

### DISCUSSION THREAD

**Discussion by Benny-21** [games647/FastLogin#713] [Q&A] [answered]
Hi everyone, I'm having a problem with the "/premium" feature. Basically, after the players use it, they still have to do the login as a normal player without premium (of course i made sure that this player were premium). I'm using "Userlogin" plugin and Fastlogin.
Thank you for your help!

**Comment by TuxCoding** [comment, marked as answer]
Please open a ticket (`bug` here), where you add the server log. It really helps find the source of this issue.
